### ISSUE TITLE
KeyError: 'fields' running in terminal

### ISSUE THREAD

**Issue by jalbertbowden** [openaddresses/pyesridump#24]
Running pyesridump on this point: 
http://apps.fs.fed.us/arcx/rest/services/EDW/EDW_InventoriedRoadlessAreas2001IdCo_01/MapServer  
getting this back in terminal:  
"j.albertbowden@sunlight_desktop:~/desktop/pyesridump-master $ esri2geojson http://apps.fs.fed.us/arcx/rest/services/EDW/EDW_InventoriedRoadlessAreas2001IdCo_01/MapServer edw-inventoriedroadlessarea2001idco01  
2016-09-23 09:06:54,652 - cli.esridump - INFO - Source does not support feature count  
Traceback (most recent call last):  
  File "/Library/Frameworks/Python.framework/Versions/2.7/bin/esri2geojson", line 11, in <module>  
    load_entry_point('esridump==1.4.0', 'console_scripts', 'esri2geojson')()  
  File "build/bdist.macosx-10.6-intel/egg/esridump/cli.py", line 100, in main  
  File "build/bdist.macosx-10.6-intel/egg/esridump/dumper.py", line 244, in __iter__  
  File "build/bdist.macosx-10.6-intel/egg/esridump/dumper.py", line 129, in _find_oid_field_name  
KeyError: 'fields'"  
In the README.md I see that it doesn't map 100%, so I'm assuming this is data is of really bad quality, but I'm not sure and am stumped as to how to even check.  
But more importantly, I'm stumped as to which direction to move in towards resolving this, if possible.  
I did see esri-dump while trying to debug this, and debated using it...but I didn't want to bail on python.
tl;dr: it really seems to me that this is just old, bad data.  
Any help is greatly appreciated!

**Comment by iandees** [comment]
The tool expects a layer, not a whole map server URL. I should definitely detect that sort of thing and give a better error message when this happens.

In your particular case, use this URL to the single layer in the MapServer you linked to:
```
http://apps.fs.fed.us/arcx/rest/services/EDW/EDW_InventoriedRoadlessAreas2001IdCo_01/MapServer/0
```

I got to that by viewing the URL you linked to in a browser and seeing that there's a single layer in the list and clicking on that layer.

**Comment by jalbertbowden** [comment]
LOL...apologies! I totally noticed the vars in the example URLs, not sure why I didn't put that together. Thanks @iandees! That worked!  
More importantly, I've been searching for a tool like this forever!
Immediately going into my opening up data toolkit utility belt.
